src/commands.c

```diff
@@ -4414,6 +4421,29 @@ const char carena_um_init[] =
 	"k_mode 2\n"
 ;
 
+const char tot_um_init[] =
+	"deathmatch 4\n"
+	"dmm4_invinc_time -1\n"
+	"dq 0\n"
+	"dr 0\n"
+	"k_bzk 0\n"
+	"k_exttime 0\n"
+	"k_fb_enabled 1\n"
+	"k_lockmax 0\n"
+	"k_lockmin 0\n"
+	"k_maxclients 9\n"
+	"k_membercount 0\n"
+	"k_mode 3\n"
+	"k_overtime 0\n"
+	"k_pow 1\n"
+	"k_spw 1\n"
+	"k_tot_mode 1\n"
+	"maxclients 9\n"
+	"sv_antilag 2\n"
```

Comment:
didn't you talk about this on discord? maybe go over this list of settings and see if it makes sense.